app/Policies/EventPolicy.php

```diff
@@ -107,7 +107,11 @@ public function removeMember(User $user, Event $event, Member $member)
 
     public function viewParticipants(User $user, Event $event)
     {
-        return $this->showAdvanced($user, $event) || ($user->hasCapability('event::show::participating') && $event->hasUser($user));
+        return $this->showAdvanced($user, $event) || (
+            $user->hasCapability('event::show::participating' &&
+            $event->hasUser($user) &&
+            $event->participants()->find($user->profile)->pivot->geplaatst)
```

Comment:
This does poef. Wanneer je een member bent.
Je moet nog ergens checken dat je user een participant is.

![image](https://user-images.githubusercontent.com/2979104/163465155-0798f440-650f-4b62-8e00-1376d161962c.png)

Repro: koppel dkrijgsman aan nieuwjaarskamp , login als hem en ga naar nieuwjaarskamp